README.md

```diff
@@ -258,16 +271,21 @@ channel; please see our [security policy][security] for details.
 [error-prone-orig-repo]: https://github.com/google/error-prone
 [error-prone-pull-3301]: https://github.com/google/error-prone/pull/3301
 [github-actions-build-badge]: https://github.com/PicnicSupermarket/error-prone-support/actions/workflows/build.yaml/badge.svg
-[github-actions-build-master]: https://github.com/PicnicSupermarket/error-prone-support/actions/workflows/build.yaml?query=branch%3Amaster
+[github-actions-build-master]: https://github.com/PicnicSupermarket/error-prone-support/actions/workflows/build.yaml?query=branch:master&event=push
 [google-java-format]: https://github.com/google/google-java-format
 [idea-288052]: https://youtrack.jetbrains.com/issue/IDEA-288052
 [license-badge]: https://img.shields.io/github/license/PicnicSupermarket/error-prone-support
 [license]: https://github.com/PicnicSupermarket/error-prone-support/blob/master/LICENSE.md
 [maven-central-badge]: https://img.shields.io/maven-central/v/tech.picnic.error-prone-support/error-prone-support?color=blue
 [maven-central-search]: https://search.maven.org/artifact/tech.picnic.error-prone-support/error-prone-support
 [maven]: https://maven.apache.org
+[openssf-best-practices-badge]: https://bestpractices.coreinfrastructure.org/projects/7199/badge
+[openssf-best-practices-checklist]: https://bestpractices.coreinfrastructure.org/projects/7199
+[openssf-scorecard-badge]: https://img.shields.io/ossf-scorecard/github.com/PicnicSupermarket/error-prone-support?label=openssf%20scorecard
+[openssf-scorecard-report]: https://api.securityscorecards.dev/projects/github.com/PicnicSupermarket/error-prone-support
```

Comment:
Is there a better link that has a UI? As this returns JSON. (Have personally not seen this on other projects, so not sure what's common here)

Reply:
I was wondering the same thing, but modeled this after what [the authors do themselves](https://github.com/ossf/scorecard). It [looks like](https://github.com/ossf/scorecard#public-data) so far there was no effort to build a proper UI. (A subset of the findings are reported [here](https://github.com/PicnicSupermarket/error-prone-support/security/code-scanning), but that view is not specific to OpenSSF Scorecard, and doesn't seem to be publicly accessible.)

Reply:
In the [launch article](https://www.endorlabs.com/blog/introducing-the-openssf-scorecard-api), they mention [tensorflow](https://github.com/tensorflow/tensorflow) and [flutter](https://github.com/flutter/flutter). Both have the badge, where tensorflow does the same as here (link to the api), whereas flutter links to https://deps.dev/project/github/flutter%2Fflutter. This is actually a nice overview! 

However, searching for `PicnicSupermarket/error-prone-support` - although found - leads to https://deps.dev/project/github/picnicsupermarket%2Ferror-prone-support, with no info 😕

Reply:
Ah, interesting! The Flutter page says:
> Project metadata as of April 15, 2023.

Maybe it doesn't regenerate very frequently. Since we should anyway finalize #575 first, let's see what happens over the next ~48 hours :eyes:

Reply:
Still 404... :shrug:

Reply:
That's unfortunate. Hoped that it would've worked in the mean time. Will approve.